vali_objects/utils/position_manager.py

```diff
@@ -299,14 +299,21 @@ def apply_order_corrections(self):
                 bt.logging.info(f"Resetting hotkey {miner_hotkey}")
                 n_corrections += 1
                 unique_corrections.update([p.position_uuid for p in positions])
-                for pos in positions:
-                    self.delete_position_from_disk(pos)
-                    self._refresh_challengeperiod_in_memory()
-                    if miner_hotkey in self.challengeperiod_testing:
-                        self.challengeperiod_testing.pop(miner_hotkey)
-                    if miner_hotkey in self.challengeperiod_success:
-                        self.challengeperiod_success.pop(miner_hotkey)
-                    self._write_challengeperiod_from_memory_to_disk()
+                #for pos in positions:
+                #    self.delete_position_from_disk(pos)
+                self._refresh_challengeperiod_in_memory()
+                if miner_hotkey in self.challengeperiod_testing:
+                    self.challengeperiod_testing.pop(miner_hotkey)
+                if miner_hotkey in self.challengeperiod_success:
+                    self.challengeperiod_success.pop(miner_hotkey)
+                self._write_challengeperiod_from_memory_to_disk()
+
+                perf_ledgers = self.perf_ledger_manager.load_perf_ledgers_from_disk()
+                print('n perf ledgers before:', len(perf_ledgers))
+                perf_ledgers_new = {k:v for k,v in perf_ledgers.items() if k != miner_hotkey}
+                print('n perf ledgers after:', len(perf_ledgers_new))
```

Comment:
I saw that we're using bt.logging.info for logging. We might want to use that instead of print statements. Looks like these are for debugging purposes.